source/templates.tex

```diff
@@ -8325,7 +8325,7 @@
 is compared with an actual type (call it
 \tcode{A}),
 and an attempt is made to find template argument values (a type for a type
-parameter, a value for a non-type parameter, or a template for a
+parameter, a value for a constant template parameter, or a template for a
 template parameter) that will make
```

Comment:
The inconsistent use of "template parameter" versus "parameter" following the change is very confusing.

```suggestion
template parameter, a value for a constant template parameter, or a template for a
template template parameter) that will make
```